src/app/core/upload/upload.service.ts

```diff
@@ -3,7 +3,8 @@ import { Observable } from 'rxjs/Observable';
 import { ConnectableObservable } from 'rxjs/observable/ConnectableObservable';
 import { Subscriber } from 'rxjs/Subscriber';
 import 'rxjs/add/operator/publish';
-import Evaporate from 'evaporate';
+declare var require: any;
```

Comment:
no, if imported it throws an error about the constructor, both this version and the version we were on since this upgrade. `require` seems to work though

Reply:
+1